packages/shared/src/write-archive/archive-file.ts

```diff
@@ -100,7 +100,7 @@ export class ArchiveFile {
 
   private removeSpecialChars(pathname: string) {
     // The storybook server seems to have a problem with percents in file names
-    return pathname.replace(/[%]/g, '');
+    return pathname.replace(/[%]|[:]/g, '');
```

Comment:
In addition to the Unit tests you've added, let's add an e2e test case for an asset with the funky `@fs/c:/` URL (for both Playwright and Cypress) so we confirm things render as expected. 

Here's how we e2e-tested the percents:
* [e2e test](https://github.com/chromaui/chromatic-e2e/blob/main/packages/playwright/tests/archiving-assets.spec.ts#L48-L50)
* [HTML file](https://github.com/chromaui/chromatic-e2e/blob/main/test-server/fixtures/asset-paths/percents.html) for the test case

(You'd also add a [test case](https://github.com/chromaui/chromatic-e2e/blob/main/packages/cypress/tests/cypress/e2e/archiving-assets.cy.ts#L41-L44) for Cypress)

Reply:
This is great! Adding now